### PR TITLE
#464 Add connection model-model and shunt single association

### PR DESCRIPTION
**Pull request by AdrienBlassiau**
**Checklist before requesting a review**

*use `'[x]'` to check the checkboxes, or submit the PR and then click the checkboxes*

- [x] unit tests and non regression tests were added (update of inputs, outputs, SA, algos)
- [x] main documentation was updated (update of input/output file)
- [ ] if this commit targets a release branch: the corresponding milestone was added in the ticket and in this PR

Related to https://github.com/dynawo/dynaflow-launcher/issues/464

### CODE REVIEW

File: documentation/configuringDynaflowLauncher/DynaflowLaunchersInputFiles.tex [hunk 1]
```diff
@@ -140,6 +140,10 @@ \subsection{Configuration file}
 & & \small{generator transformers are}  &   \\
 \rowcolor{white}
 \multirow{-3}{*}{\small{TfoVoltageLevel}}&  \multirow{-3}{*}{\small{real}} & \small{considered to be in the iidm file}  &  \multirow{-3}{*}{\small{100}} \\
+\rowcolor{gray!10}
+\small{AssemblingPath} & \small{string} & \small{Path to the assembling file} & \small{nothing} \\
```
Comment (cossartque) [inline documentation/configuringDynaflowLauncher/DynaflowLaunchersInputFiles.tex:144]: Already implemented

File: documentation/functionalDoc/functionalDoc.tex [hunk 2]
```diff
@@ -633,4 +633,89 @@ \section{Solver}
 
 \lstinputlisting[language=XML,title=\Dynawo Simplified Solver configuration]{../resources/syntaxExample/solver.par}
 
+\section{Assembling and Setting files}
```
Comment (cossartque) [inline documentation/functionalDoc/functionalDoc.tex:636]: Check if not already implemented

**Comment by cossartque** [comment]
Could you also resolve the conflicts ? Thanks

**Review by rosiereflo — CHANGES_REQUESTED** [review]

File: etc/xsd/assembling.xsd [hunk 3]
```diff
@@ -97,10 +112,14 @@ SPDX-License-Identifier: MPL-2.0 -->
     <xs:attribute name="voltageLevel" use="required" type="xs:string"/>
   </xs:complexType>
 
-  <xs:complexType name="Shunt">
+  <xs:complexType name="MultipleShunts">
     <xs:attribute name="voltageLevel" use="required" type="xs:string"/>
   </xs:complexType>
 
+  <xs:complexType name="SingleShunt">
+    <xs:attribute name="name" type="xs:string"/>
```
[the remark this answers is not in the document]
Comment (jlbouchot) [reply etc/xsd/assembling.xsd:120]: I believe this is okay -- unless my understanding is far fro, the truth. In a multipleAssociation case, one would want to connect *all* shunts connected to a given voltage level whereas in the singleAssociation case, one would want to target a specific shunt, independently of the other shunts connected to the same voltage level. 

This raises the (potential) question though of "What to do if a multipleAssociation is mentioned for a given voltage level and one of the shunts has a specific single association too?" I'm guessing it doesn't matter much, but I could be wrong.

**Review by rosiereflo — APPROVED** [review]

**Pull request merged**